### PR TITLE
HomeWork OOP

### PR DESCRIPTION
**Pull request by ViktoriiaSvidovska**

### CODE REVIEW

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working");
+    }
+
```
Comment (uliana-andrushchuk) [inline src/main/java/core/basesyntax/Bulldozer.java:13]:
```suggestion
```
remove empty line

File: src/main/java/core/basesyntax/Excavator.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
```
Comment (uliana-andrushchuk) [inline src/main/java/core/basesyntax/Excavator.java:4]:
```suggestion
```

File: src/main/java/core/basesyntax/Machine.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+        public abstract void doWork();
+
+        public abstract void stopWork();
+    }
```
Comment (uliana-andrushchuk) [inline src/main/java/core/basesyntax/Machine.java:7]: Please, fix formatting

**Review by uliana-andrushchuk — CHANGES_REQUESTED** [review]
Good job, but please fix formatting

**Review by olekskov — CHANGES_REQUESTED** [review]
Build is failed. Before pushing changes please use 'mvn clean package' command locally and fix issues highlighted by maven in case of failed build

**Review by kozhukhovsky — APPROVED** [review]

**Pull request closed without merging**